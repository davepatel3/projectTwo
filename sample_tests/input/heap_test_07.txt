5437
create_file
61
GGJiLxx6f
0
rename
1
Ci4JboBH6yQUrx8
remove
1
create_directory_root
sOkq4b
create_file
22
wII2V
1
create_directory
6Xmu ml6Zu9
1
create_directory
6BhCISTJ5_7
1
create_file
156
6BhCISTJ5_7
1
create_link_root
1
Mp5VE7j8rQVTIau 
create_directory
7zbr5g6I
3
create_link
4
SZmdA
4
create_file
21
2Gc6oUaVjx
1
remove
8
create_directory
RyPCkYnb2CSZ0xV
3
get_name
7
create_file
30
C1uqz CLYhxGw4nAZC
1
create_file
8
X19WGPMtXJnb
3
get_largest_file_handle
create_link
7
N71_Omn80D-
4
get_absolute_path
8
get_handle
/sOkq4b/6BhCISTJ5_7/N71_Omn80D-
remove
10
create_link
3
LseSd3IHTHmPog6TJ
3
rename
11
P1pI 5ZaCbL0DxK_
create_link
10
MTjQ5m
8
rename
11
6cM--9A
get_name
12
remove
3
get_largest_file_handle
get_largest_file_handle
create_link
4
ZFrPVAPb7 Jp6aq
1
create_file
21
gxEkIx3aVVVqLO6ptzo
6
create_link_root
8
-gbhVECsHbrueKFNgt0h
create_link
10
DyUW9gLlbh
3
create_link
5
fflU
1
create_file
35
30vcOy
8
get_largest_file_handle
create_link
8
Bygyxm
6
rename
1
6AvxsIv_g
create_directory
2MGWlxvv86QYxFx
4
get_name
18
rename
7
dXahhzxPQJqPSI
get_name
1
get_absolute_path
16
create_directory
4S2YWWsGPLLWpaoYEvRk
8
create_directory
kVzwE5fDV_m8eAIsGm0
8
rename
5
V8wiwmWzdzUN
create_file
42
Kw48QRe1TMFomuRzv
21
create_file
249
4EAC2Kl
8
get_file_size
11
create_link
13
9dSs9s
1
print_layout
get_available_size
get_file_size_path
/6AvxsIv_g/6Xmu ml6Zu9/7zbr5g6I/gxEkIx3aVVVqLO6ptzo
rename
22
hrFNm
create_directory
yaEM4Nsi
6
create_file
50
CrnPIyAlOdz4G123b
22
create_directory
uGrpI5j4
3
get_absolute_path
18
get_available_size
get_name
28
create_link
5
n5N6Y
21
create_directory
iAAWCST2C01pNxMRxS8B
3
create_directory
DyUW9gLlbh
3
rename
30
yPUx
get_largest_file_handle
get_name
24
create_link
14
GPF0
20
create_file
10
orhRS5d9c
28
create_file
11
Jrsxevr
1
remove
22
rename
8
YxnP-7
rename
11
y mnl65eH8g0Y
create_link
32
fgy9JvhBOSpl
21
create_file
158
LhKqglwJvF3cpPV
0
create_file
134
oAO1Ln3FXTWMjrWLOWwx
8
create_directory
odMSybf33I3W2ag99
28
remove
26
exist
15
get_file_size
27
create_file
133
gM Vs1C3
22
get_available_size
get_name
14
create_file
66
xpaVI8m9vyLMijx
1
create_link
14
IB_Mf8c73jRtfWJl
3
get_file_size
26
create_link
20
BlQJBOVQUkC_tGy4S
6
create_directory
jKY0bqaCOVGqj
0
create_directory
V j6qOLXg6cK7qOuTR
20
create_link
24
 yaWiNzNwnBY
3
get_name
2
get_name
8
create_file
147
B7APpu3K69Vf3GDGfR
1
create_directory
KmYq8il9iV
22
get_available_size
create_link
5
lbIX20rchRnrG3
21
create_directory
x- N48qY5
37
rename
13
RVEBuQJtXl
create_file
4
lZBAdIT
30
create_file
230
gCDM_sG32p0sfC
0
create_link
20
cfDkPVdunjgSit1mf
20
remove
19
create_directory
2zQXq8Ztba8
6
create_file
6
SBKgfS9A5uTP0x1GY87
4
follow
13
get_available_size
quit
