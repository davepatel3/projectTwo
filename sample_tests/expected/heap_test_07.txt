create_file
1
rename
remove
create_directory_root
1
create_file
2
create_directory
3
create_directory
4
create_file
File already exists!
create_link_root
5
create_directory
6
create_link
7
create_file
8
remove
create_directory
8
get_name
SZmdA
create_file
9
create_file
10
get_largest_file_handle
9
create_link
11
get_absolute_path
/sOkq4b/6Xmu ml6Zu9/RyPCkYnb2CSZ0xV
get_handle
11
remove
create_link
10
rename
create_link
12
rename
get_name
MTjQ5m
remove
get_largest_file_handle
9
get_largest_file_handle
9
create_link
13
create_file
14
create_link_root
15
create_link
16
create_link
17
create_file
18
get_largest_file_handle
18
create_link
19
rename
create_directory
20
get_name
30vcOy
rename
get_name
6AvxsIv_g
get_absolute_path
/6AvxsIv_g/6Xmu ml6Zu9/DyUW9gLlbh
create_directory
21
create_directory
22
rename
create_file
23
create_file
24
get_file_size
Invalid handle!
create_link
25
print_layout
[D]6AvxsIv_g
	[F]wII2V (size = 22)
	[D]6Xmu ml6Zu9
		[D]7zbr5g6I
			[F]gxEkIx3aVVVqLO6ptzo (size = 21)
			[L]Bygyxm [->/6AvxsIv_g/6Xmu ml6Zu9/RyPCkYnb2CSZ0xV]
		[D]RyPCkYnb2CSZ0xV
			[L]MTjQ5m [->/6AvxsIv_g/6Xmu ml6Zu9]
			[F]30vcOy (size = 35)
			[D]4S2YWWsGPLLWpaoYEvRk
				[F]Kw48QRe1TMFomuRzv (size = 42)
			[D]kVzwE5fDV_m8eAIsGm0
			[F]4EAC2Kl (size = 249)
		[L]LseSd3IHTHmPog6TJ [->/6AvxsIv_g/6Xmu ml6Zu9]
		[L]DyUW9gLlbh [->/6AvxsIv_g/6Xmu ml6Zu9]
	[D]6BhCISTJ5_7
		[L]dXahhzxPQJqPSI [->/6AvxsIv_g/6BhCISTJ5_7]
		[L]6cM--9A [->/6AvxsIv_g/6BhCISTJ5_7]
		[D]2MGWlxvv86QYxFx
	[F]C1uqz CLYhxGw4nAZC (size = 30)
	[L]ZFrPVAPb7 Jp6aq [->/6AvxsIv_g/6BhCISTJ5_7]
	[L]fflU [->/6AvxsIv_g]
	[L]9dSs9s [->/6AvxsIv_g/6BhCISTJ5_7]
[L]V8wiwmWzdzUN [->/6AvxsIv_g]
[L]-gbhVECsHbrueKFNgt0h [->/6AvxsIv_g/6Xmu ml6Zu9/RyPCkYnb2CSZ0xV]

get_available_size
5038
get_file_size_path
21
rename
create_directory
26
create_file
27
create_directory
28
get_absolute_path
/6AvxsIv_g/6Xmu ml6Zu9/RyPCkYnb2CSZ0xV/30vcOy
get_available_size
4988
get_name
uGrpI5j4
create_link
29
create_directory
30
create_directory
Directory already exists!
rename
get_largest_file_handle
24
get_name
4EAC2Kl
create_link
31
create_file
32
create_file
33
remove
rename
rename
create_link
34
create_file
35
create_file
36
create_directory
37
remove
exist
1
get_file_size
50
create_file
26
get_available_size
4542
get_name
gxEkIx3aVVVqLO6ptzo
create_file
38
create_link
39
get_file_size
133
create_link
40
create_directory
41
create_directory
42
create_link
43
get_name
wII2V
get_name
YxnP-7
create_file
44
create_directory
45
get_available_size
4329
create_link
46
create_directory
47
rename
create_file
48
create_file
49
create_link
50
remove
create_directory
19
create_file
51
follow
4
get_available_size
4089
quit
